app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -70,7 +70,7 @@ def revert_content(revision_id, relation = 'original_file')
       # @param [File, ActionDigest::HTTP::UploadedFile, Tempfile] file the file uploaded by the user.
       # @param [String] relation ('original_file')
       def update_content(file, relation = 'original_file')
-        file_actor_class.new(file_set, relation, user).ingest_file(file)
+        file_actor_class.new(file_set, relation, user).ingest_file(file, true)
```

Comment:
If this is refactored to have a `file_actor` method that instantiates the `file_actor_class`, then this method would be easier to test